MockKit/MockCGMManager.swift

```diff
@@ -329,8 +329,13 @@ public final class MockCGMManager: TestingCGMManager {
                                              foregroundContent: Alert.Content(title: "Signal Loss", body: "CGM simulator signal loss", acknowledgeActionButtonLabel: "Dismiss"),
                                              backgroundContent: Alert.Content(title: "Signal Loss", body: "CGM simulator signal loss", acknowledgeActionButtonLabel: "Dismiss"))
 
+    private let lockedMockSensorState = Locked(MockCGMState(isStateValid: true))
```

Comment:
These changes look like there are for https://tidepool.atlassian.net/browse/LOOP-3884. Consider a different PR or, at the very minimum, reference this ticket in the commits and PR.